src/pages/login/login.js

```diff
@@ -32,8 +34,24 @@ export const Login=()=>{
         }
     }
 
+    const testLogin= async()=>{
+        try {
+            const response=await axios.post('/api/auth/login',{
+                email:"adarshbalika@gmail.com",
+                password:"adarshbalika"
+            })
+            localStorage.setItem("user",response.data.encodedToken);
+            setAuth(true)
+            navigate("/")
+        } catch (error) {
+            console.log(error);
```

Comment:
you can use toastify to show error rather than logging on console

Reply:
ok will do it